src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -41,7 +41,8 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
         EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(editedPerson).build();
         EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, descriptor);
 
-        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS, Messages.format(editedPerson));
+        String expectedMessage = String.format(
+                EditCommand.MESSAGE_EDIT_PERSON_SUCCESS, Messages.formatPerson(editedPerson));
```

Comment:
Should we follow this format of `MESSAGE_ACTION_SUCCESS` or will `MESSAGE_SUCCESS` do?

Reply:
Cuz I didn't reach the Edit commands, so maybe we can formalize it use MESSAGE_SUCCESS later when we implementing EditPersonCommand and EditCompanyCommand.

Reply:
For now it works, can resolve this when someone taking the Edit part.